src/components/Slider/slider.module.scss

```diff
@@ -284,21 +347,25 @@ $small-marker-top: 0;
     // For Chrome browsers
     &::-webkit-slider-thumb {
       -webkit-appearance: none;
-      background-color: var(--slider-default-thumb-color);
+      background-color: var(--slider-thumb-color);
```

Comment:
highly unlikely anyone is actually using this yet, but are we considering some of these var name adjustments breaking change? in the future we definitely should

Reply:
Agreed, figure it's early enough from the outset of these var names to update them as all forks of the repo are Eightfold folks.

Reply:
Given that was a scoped variable (not part of varTheme), we should be okay.